src/Components/ImportContactsDialog/ImportContactsDialog.jsx

```diff
@@ -109,9 +111,12 @@ const ImportContactsDialog = ({ status, callback, onImport }) => {
 
             var all = document.getElementsByClassName("initial__modal");
             for (var i = 0; i < all.length; i++) {
-              all[i].style.display = "block";
+              all[i].style.display = "none";
             }
 
+            // Open Create NFT Dialog
+            dispatch({ type: "createnft__open" });
```

Comment:
This should be conditional, if someone is importing the contact for the 1st time after login, then we have to open create NFT popup, else if someone is accessing this after creating the NFT then we should open the contact list popup. But, this line will always open the create NFT popup after importing the contact.

Reply:
Ok

Reply:
@biswanathsahoo Resolved conflict issue.